source/common/http/headers.h

```diff
@@ -79,6 +79,7 @@ class CustomHeaderValues {
   const LowerCaseString Pragma{"pragma"};
   const LowerCaseString Referer{"referer"};
   const LowerCaseString Vary{"vary"};
+  const LowerCaseString WwwAuthenticate{"www-authenticate"};
```

Comment:
You re-define it.  It is defined in line 215.

Reply:
Whoops, I looked at the comment on my phone and didn't notice that. Fixing now